nifi-rowtodoc-processors/src/main/java/com/marklogic/nifi/processors/rowtodoc/ConvertColumnMapsToJSON.java

```diff
@@ -20,8 +24,14 @@
 	protected List<PropertyDescriptor> properties;
 	protected Set<Relationship> relationships;
 
-	// TODO Pull this from a controller service?
-	private ObjectMapper objectMapper = new ObjectMapper();
+	private ObjectMapper objectMapper;
+
+	public static final PropertyDescriptor SQL_DATE_FORMAT = new PropertyDescriptor.Builder()
+		.name("java.sql.Date format")
+		.defaultValue("yyyy-MM-dd")
+		.description("Date format for serializing instances of java.sql.Date")
+		.addValidator(Validator.VALID)
```

Comment:
I think a future enhancement would be to use a date format validator instead of just accepting the input